bbn-test-5/finality-providers/README.md

```diff
@@ -0,0 +1,591 @@
+## Introduction
+
+Finality providers are responsible for voting at a finality round on top
+of [CometBFT](https://github.com/cometbft/cometbft). Similar to any native PoS
+validator, a finality provider can receive voting power delegations from BTC
+stakers, and can earn commission from the staking rewards denominated in Babylon
+tokens.
+
+The finality provider toolset does not have any special hardware requirements
```

Comment:
This will soon change as we're working on a secure keyring setup for EOTS manager. 
We could include a note for this, as it's a very common question by collaborators.